aviary/subsystems/propulsion/test/test_prop_performance.py

```diff
@@ -36,159 +36,110 @@ def setUp(self):
         prob = self.prob
-        prob.set_val(Dynamic.Mission.ALTITUDE, 0.0, units="ft")
-        prob.set_val(Dynamic.Mission.VELOCITY, 0.0, units="knot")
-        prob.set_val(Dynamic.Mission.PROPELLER_TIP_SPEED, 800.0, units="ft/s")
-        prob.set_val(Dynamic.Mission.SHAFT_POWER, 1850.0, units="hp")
+        prob.set_val(Dynamic.Mission.ALTITUDE, [0.0, 0.0, 25000.0], units="ft")
+        prob.set_val(Dynamic.Mission.VELOCITY, [0.10, 125.0, 300.0], units="knot")
+        prob.set_val(Dynamic.Mission.PROPELLER_TIP_SPEED,
+                     [800.0, 800.0, 750.0], units="ft/s")
+        prob.set_val(Dynamic.Mission.SHAFT_POWER, [1850.0, 1850.0, 900.0], units="hp")
 
         prob.run_model()
-        self.compare_results(case_idx=0)
+        self.compare_results(case_idx_begin=0, case_idx_end=2)
 
-    def test_case_1(self):
-        # Case 1
-        prob = self.prob
-        prob.set_val(Dynamic.Mission.ALTITUDE, 0.0, units="ft")
-        prob.set_val(Dynamic.Mission.VELOCITY, 125.0, units="knot")
-        prob.set_val(Dynamic.Mission.PROPELLER_TIP_SPEED, 800.0, units="ft/s")
-        prob.set_val(Dynamic.Mission.SHAFT_POWER, 1850.0, units="hp")
-
-        prob.run_model()
-        self.compare_results(case_idx=1)
-
-    def test_case_2(self):
-        # Case 2
-        prob = self.prob
-        prob.set_val(Dynamic.Mission.ALTITUDE, 25000.0, units="ft")
-        prob.set_val(Dynamic.Mission.VELOCITY, 300.0, units="knot")
-        prob.set_val(Dynamic.Mission.PROPELLER_TIP_SPEED, 750.0, units="ft/s")
-        prob.set_val(Dynamic.Mission.SHAFT_POWER, 900.0, units="hp")
-
-        prob.run_model()
-        self.compare_results(case_idx=2)
-
-    def test_case_3(self):
-        # Case 3
-        prob = self.prob
-        options = self.options
-
-        options.set_val(Aircraft.Design.COMPUTE_INSTALLATION_LOSS,
-                        val=False, units='unitless')
-        prob.setup()
-        prob.set_val(Dynamic.Mission.INSTALLATION_LOSS_FACTOR, 0.0, units="unitless")
-        prob.set_val(Aircraft.Engine.PROPELLER_DIAMETER, 12.0, units="ft")
-        # prob.set_val(Aircraft.Nacelle.AVG_DIAMETER, 2.4, units='ft')
-        prob.set_val(Aircraft.Engine.PROPELLER_ACTIVITY_FACTOR, 150.0, units="unitless")
-        prob.set_val(Aircraft.Engine.PROPELLER_INTEGRATED_LIFT_COEFFICENT,
-                     0.5, units="unitless")
-        # prob.set_val('DiamNac_DiamProp', 0.2, units="unitless")
-        prob.set_val(Dynamic.Mission.ALTITUDE, 10000.0, units="ft")
-        prob.set_val(Dynamic.Mission.VELOCITY, 200.0, units="knot")
-        prob.set_val(Dynamic.Mission.PROPELLER_TIP_SPEED, 750.0, units="ft/s")
-        prob.set_val(Dynamic.Mission.SHAFT_POWER, 1000.0, units="hp")
-
-        prob.run_model()
-        self.compare_results(case_idx=3)
+        partial_data = prob.check_partials(
+            out_stream=None, compact_print=True, show_only_incorrect=True, form='central', method="fd",
```

Comment:
The `Hamiltonstandard()` model calls `fd` derivatives as well and doesn't specify `form`. OpenMDAO docs note that FD form is going to be based by default on the last FD form called: https://openmdao.org/newdocs/versions/latest/features/core_features/working_with_derivatives/check_partials_settings.html?highlight=check_partials

To successfully test the partials of a method that uses FD you have to use a different FD method. Thus, we need to specify `form='forward'` or `form='backward'` in  `Hamiltonstandard()`.